includes/admin/plugins.php

```diff
@@ -398,6 +397,14 @@ function perflab_render_plugin_card( array $plugin_data ): void {
 			esc_html__( 'Visit plugin site', 'default' )
 		);
 	}
+
+	if ( $availability['activated'] ) {
+		$plugin_link = perflab_get_plugin_settings_link( $plugin_data['slug'] );
+		if ( $plugin_link ) {
+			/* translators: %s is the settings URL */
+			$action_links[] = sprintf( __( '<a href="%s">Settings</a>', 'performance-lab' ), esc_url( $plugin_link ) );
```

Comment:
This isn't currently getting escaped. This should do it:

```suggestion
			$action_links[] = sprintf( '<a href="%s">%s</a>', esc_url( $plugin_link ), esc_html__( 'Settings', 'performance-lab'  ) );
```